src/components/comment/comment_timeline.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+import { CommonProps, keysOf } from '../common';
+import classNames from 'classnames';
+import { EuiIcon } from '../icon';
+
+export type EuiCommentTimelineProps = CommonProps & {
```

Comment:
```suggestion
export type EuiCommentTimelineProps = CommonProps &
  HTMLAttributes<HTMLDivElement> & {
```

and import `HTMLAttributes` from 'react'